osidb/tests/factories.py

```diff
@@ -127,7 +127,15 @@ class Meta:
             )
         ]
     )
-    meta_attr = factory.Dict({"test": "1"})
+    # valid flaw is expected to have certain meta attributes present
+    meta_attr = factory.LazyAttribute(
```

Comment:
note: If both `bz_id` and `meta_attr` parameters are passed to the factory, `bz_id` is `overwritten` by the passed `meta_attr`, is this expected?

Reply:
This is a side effect I am not sure we can easily prevent. It would be however a developer mistake. I introduced `bz_id` factory attribute as a kind of shortcut (similar to `embargoed`) which ease the work of writing the factory. The lazy attribute then makes the instance produced by the factory as we generally expect it so again easing the work for the test developer. If one needs some more complicated `meta_attr`, (s)he simply has to write it all by hand and then it makes no sense to provide `bz_id` attribute also.

Reply:
Understandable. Seems ok.